packages/flutter_tools/lib/src/test/flutter_platform.dart

```diff
@@ -268,6 +270,8 @@ class _FlutterPlatform extends PlatformPlugin {
           subprocessActive = false;
           final String message = _getErrorMessage(_getExitCodeMessage(exitCode, 'before connecting to test harness'), testPath, shellPath);
           controller.sink.addError(message);
+          // Waiting below.
+          // ignore: unawaited_futures
```

Comment:
as above, these comments should all just be on one line with the code they impact.